src/Examples/Advanced/CustomProperties/Hydrocarbon_Processing_example.ipynb

```diff
@@ -0,0 +1,1078 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Hydrocarbon Processing with cubic equation of state case study"
```

Comment:
One general comment on this example - it takes a really long time to run for me. I think we need to consider how we are going to handle this, as my guess is that it would seriously impact the run time of our integration tests.

Reply:
I will include the json file with the initialization to avoid the time it takes. I will leave the sensitivity analysis but I will comment it out and add the plots as images so people can see how I do this but avoid the run time.